lib/puma/server.rb

```diff
@@ -58,6 +59,7 @@ class Server
     def initialize(app, events=Events.stdio, options={})
       @app = app
       @events = events
+      @debug_logger = DebugLogger.stdio
```

Comment:
I think ideally no one would access this interface directly, but instead would access it indirectly through a method on `Events`, like parse_error.

Reply:
Yup, removed debug_logger from the server and replaced its invokes with wrappers from the events class